### PR TITLE
configure phoenix apps

### PR DESCRIPTION
**Pull request by butonic**
configures the apps and external apps for the
ocis & ocis-simple build by conditionally including a
different configurePhoenix function

### CODE REVIEW

File: pkg/command/phoenix_ocis.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	cfg.Phoenix.Log.Pretty = cfg.Log.Pretty
+	cfg.Phoenix.Log.Color = cfg.Log.Color
+	cfg.Phoenix.Tracing.Enabled = false
+	cfg.Phoenix.HTTP.Addr = "localhost:9100"
+	cfg.Phoenix.HTTP.Root = "/"
+	cfg.Phoenix.Phoenix.Config.Apps = []string{
+		"draw-io",
+		"files",
+		"markdown-viewer",
+		"medie-viewer",
```
Comment (DeepDiver1975) [inline pkg/command/phoenix_ocis.go]: media

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pkg/command/phoenix_simple.go  1
- pkg/command/phoenix_ocis.go  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=4798872&bid=15971214)

File: go.mod [hunk 2]
```diff
@@ -14,10 +14,12 @@ require (
...
-	github.com/owncloud/ocis-phoenix v0.0.0-20191224105637-984f99d54c4c
+	github.com/owncloud/ocis-phoenix v0.0.0-20200109152733-819c7bec7b70
 	github.com/owncloud/ocis-pkg v1.2.1-0.20191217084055-eab942498596
 	github.com/owncloud/ocis-reva v0.0.0-20191220070520-aabd71f6d16b
 	github.com/owncloud/ocis-webdav v0.0.0-20191227185841-c5c77e241249
 	github.com/spf13/viper v1.6.1
 	go.opencensus.io v0.22.2
 )
+
+replace github.com/owncloud/ocis-phoenix => ../ocis-phoenix
```
Comment (DeepDiver1975) [inline go.mod]: why?

Comment (butonic) [reply go.mod]: good catch

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pkg/command/phoenix_simple.go  1
- pkg/command/phoenix_ocis.go  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=4798872&bid=15971214)

**Pull request merged**